### PR TITLE
Build out Phone kit

### PR DESCRIPTION
**Pull request by eabousaif**

### CODE REVIEW

File: app/pb_kits/playbook/pb_phone/docs/_phone_default.html.erb [hunk 1]
```diff
@@ -0,0 +1,4 @@
+<%= pb_rails("phone", props: {
+  icon: "mobile-android",
+  number: "(349) 185-9988"
```
Comment (jasoncypret) [inline app/pb_kits/playbook/pb_phone/docs/_phone_default.html.erb]: Can you add a formatted in the kit class that will take a number, strip out any non-numeric characters, then use `number_to_phone(raw_numbers, area_code: true)` to make the number look good?

Comment (eabousaif) [reply app/pb_kits/playbook/pb_phone/docs/_phone_default.html.erb]: Add a number formatter to the number method in phone.rb

File: app/pb_kits/playbook/pb_phone/docs/_phone_default.html.erb [hunk 2]
```diff
@@ -0,0 +1,4 @@
+<%= pb_rails("phone", props: {
+  icon: "mobile-android",
+  number: "3491859988"
```
Comment (jasoncypret) [inline app/pb_kits/playbook/pb_phone/docs/_phone_default.html.erb]: I tried passing a string that might come through nitro `349-185-9988` & it breaks.

We want to first strip any non-numeric characters. Then after the number is only numbers, use this to format the number `number_to_phone(your_formatted_number, area_code: true)`

Comment (jasoncypret) [reply app/pb_kits/playbook/pb_phone/docs/_phone_default.html.erb]: You could use:
`"(627)222-2777".delete('^0-9')`
or 
`"(627)222-2777".tr('^0-9', '')`

Comment (eabousaif) [reply app/pb_kits/playbook/pb_phone/docs/_phone_default.html.erb]: Tried with different formats and it worked fine

**Pull request merged**